### PR TITLE
Detect keyboard and mouse and pass them to user-VM

### PR DESCRIPTION
**Pull request by jsvapiav**
jira: HYPR-435

Signed-off-by: Ivan Kuznetsov <jsvapiav@gmail.com>

### CODE REVIEW

File: recipes-devtools/qemu/qemu-sel4-helpers/qemu-sel4-gui [hunk 1]
```diff
@@ -28,6 +43,8 @@ export GUEST_GUI_ARGS="${GUEST_GUI_ARGS:-\
 	$GUEST_DISPLAY \
 	$GUEST_DISPLAY_DEVICE \
 	$GUEST_DISPLAY_ARGS \
+        $GUEST_MOUSE_DEV \
+        $GUEST_KBD_DEV \
```
Comment (elmankku) [inline recipes-devtools/qemu/qemu-sel4-helpers/qemu-sel4-gui]: Seems like we have mixed tab & spaces in this file

**Review by elmankku — COMMENTED** [review]
Fix tabs/spaces to either way and good to go :+1:

File: recipes-devtools/qemu/qemu-sel4-helpers/qemu-sel4-gui (hunk 1 above)
Comment (jsvapiav) [reply recipes-devtools/qemu/qemu-sel4-helpers/qemu-sel4-gui]: Yep, my fault I have tab -> 4 spaces conversion by vim. Will fix it.

Comment (jsvapiav) [reply recipes-devtools/qemu/qemu-sel4-helpers/qemu-sel4-gui]: Fixed.

**Pull request merged**